BIPs/BIP-162/ethereumTreasury.json

```diff
@@ -0,0 +1,151 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1674765376433,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.11.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
```

Comment:
The DAO Treasury on Ethereum eth:0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f will send the following assets to the LM Multisig: eth:0xc38c5f97B34E175FFd35407fc91a937300E33860,